### ISSUE TITLE
SPIKE: Investigation v3 updates to eRSD

### ISSUE THREAD

**Issue by m-goggins** [CDCgov/phdi#2742]
- What are the differences between v2 and v3? 
** New conditions?
** Updates to conditions?
** Additions, deletions?

- Should we start the QC DB from v3?

**Comment by m-goggins** [comment]
tl;dr: The release logs are cumulative and not amazing. We should start with v3 because it complies with eICR 1.1 and eICR 3.1. We should also consider how we want to display changes to users and how we will extract those changes from future changelogs.  

Release notes and change log can be found here: [https://ersd.aimsplatform.org/#/home](https://ersd.aimsplatform.org/#/home), though the documentation is somewhat lacking. For example, it says for both versions 2 and 3 that there are 213 reportable conditions + 13 new conditions (and the "new" conditions are the same), which makes me think that they are not exactly updating all of the documentation each time. 

### Changes between v2 and v3
- The primary change between v2 to v3 is that eRSD Version 3 includes the queries necessary for producing both CDA eICR R1.1 and CDA eICR R3.1 reports whereas v2 only includes the codes for eICR R1.1. 
- Value sets were updated, as well as changes/updates to many individual codes
- Mostly additive changes

### Notes for interpreting the changelogs:
- grouping list tells you which value sets were updated
- code lists tells you which individual codes were updates
- Codes can be active, inactive (deprecated; should not be used), remapped (mapped to new code in VSAC but no indication of what the new code is :( ), removed. Some codes are listed as active but the description is "removed" so I'm not sure how to interpret that...

**Comment by m-goggins** [comment]
[RCTC_Release (2024-06-28).xlsx](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNUZKQnc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--a3591f1c7589cca3f3cb3e8e1b1948aaa5171b5e/RCTC_Release%20%282024-06-28%29.xlsx)
[RCTC_Change_Log (2024-06-28).xlsx](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNUpKQnc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--a4125b20509ddf5840384452181497bd9e67369d/RCTC_Change_Log%20%282024-06-28%29.xlsx)
